CHANGELOG.md

```diff
@@ -10,7 +10,7 @@
   - `test:references`: fails if typescript references are out of sync.
   - `prepare:references`: updates typescript references, if required.
 - [repo] the `prepare` script now updates typescript references.
-- [core] From now on, downstream projects can refine where the configuration files (such as `settings.json`, `keymaps.json`, `recentworkspace.json`, etc.) will be stored by Theia. [#4488](https://github.com/eclipse-theia/theia/pull/4488)  
+- [core] From now on, downstream projects can refine where the configuration files (such as `settings.json`, `keymaps.json`, `recentworkspace.json`, etc.) will be stored by Theia. [#4488](https://github.com/eclipse-theia/theia/pull/4488)
```

Comment:
It was on purpose. In markdown, double whitespace is a [linebreak](https://gist.github.com/shaunlebron/746476e6e7a4d698b373). Please revert. If it was automatic, we have an incorrect editorconfig somewhere.

Reply:
TIL: it seems [`\` should also work](https://gist.github.com/shaunlebron/746476e6e7a4d698b373#gistcomment-2349833). If so, please use it. Thanks!

Reply:
> It was on purpose. In markdown, double whitespace is a [linebreak](https://gist.github.com/shaunlebron/746476e6e7a4d698b373). Please revert. If it was automatic, we have an incorrect editorconfig somewhere.

I reverted the change, the trimming of whitespaces was done automatically.
Unfortunately, until we find the cause it looks like this might happen each time the file is modified.

Reply:
It comes from the following VS Code default preference:

<img width="411" alt="Screen Shot 2020-02-26 at 12 51 33 PM" src="https://user-images.githubusercontent.com/40359487/75372699-e3e51000-5896-11ea-86a0-3134cc08b9ec.png">

Reply:
Thanks for the update; `\` is better. 👍

Reply:
> Thanks for the update; `\` is better. 👍

No problem! Thank you for catching it and providing a suggestion :)